x/market/client/cli/txDeal.go

```diff
@@ -70,6 +71,35 @@ func CmdSellData() *cobra.Command {
 	return cmd
 }
 
+func CmdDeactivateDeal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "deactivate-deal [dealId]",
+		Short: "deactivate-deal",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return nil
```

Comment:
```suggestion
				return err
```
It seems that the error return is missing.